instance/models/openedx_appserver.py

```diff
@@ -456,6 +457,10 @@ def accepts_ssh_commands():
             """ Does server accept SSH commands? """
             return self.server.status.accepts_ssh_commands
 
+        def heartbeat_active():
+            """Is this server's /heartbeat returning 200s?"""
```

Comment:
Note that this comment is inaccurate: see [here](https://github.com/requests/requests/blob/e4fc3539b43416f9e9ba6837d73b1b7392d4b242/requests/models.py#L689-L695).

Reply:
@UmanShahzad you think we should check specifically for 200 here? or just update the comment?

Reply:
@rocioar I wanted to mimic what HAProxy does with `option httpchk`, and [according to the documentation](https://github.com/haproxy/haproxy/blob/master/doc/configuration.txt#L3785-L3786), it seems to do  what `.ok` does, so probably just change the comment.